fishtest/fishtest/templates/nn_upload.mak

```diff
@@ -11,8 +11,7 @@
     </p>
     <p>
     By uploading you license your network under a
-    <a href="https://creativecommons.org/share-your-work/public-domain/cc0/">CC0</a> or
-    <a href="https://creativecommons.org/licenses/by/4.0/">CC BY 4.0</a> license.
+    <a href="https://creativecommons.org/share-your-work/public-domain/cc0/">CC0</a>.
```

Comment:
Maybe we should add one more sentence.
`You are the author of this network or have obtained the network with a CC0 license.`

Reply:
Done